crates/re_selection_panel/src/defaults_ui.rs

```diff
@@ -139,26 +140,68 @@ pub fn defaults_ui(ctx: &ViewContext<'_>, space_view: &SpaceViewBlueprint, ui: &
 }
 
 #[allow(clippy::too_many_arguments)]
-pub fn add_new_default(
+fn add_new_default(
     ctx: &ViewContext<'_>,
     query: &LatestAtQuery,
     ui: &mut egui::Ui,
     component_to_vis: &BTreeMap<ComponentName, ViewSystemIdentifier>,
-    active_overrides: &BTreeSet<ComponentName>,
+    active_defaults: &BTreeSet<ComponentName>,
     defaults_path: &EntityPath,
 ) {
-    let remaining_components = component_to_vis
-        .keys()
-        .filter(|c| !active_overrides.contains(*c))
+    let mut disabled_reason = None;
+    if component_to_vis.is_empty() {
+        disabled_reason = Some("No components to visualize".to_owned());
+    }
+
+    let mut component_to_vis = component_to_vis
+        .iter()
+        .filter(|(component, _)| !active_defaults.contains(*component))
         .collect::<Vec<_>>();
 
-    let enabled = !remaining_components.is_empty();
+    if component_to_vis.is_empty() && disabled_reason.is_none() {
+        disabled_reason = Some("All components already have active defaults".to_owned());
+    }
+
+    {
+        // Make sure we have editors. If we don't, explain to the user.
+        let mut missing_editors = vec![];
+
+        component_to_vis.retain(|(component, _)| {
+            let component = **component;
 
-    ui.add_enabled_ui(enabled, |ui| {
-        let mut opened = false;
+            // If there is no registered editor, don't let the user create an override
+            // TODO(andreas): Can only handle single line editors right now.
+            let types = ctx
+                .viewer_ctx
+                .component_ui_registry
+                .registered_ui_types(component);
+
+            if types.contains(ComponentUiTypes::SingleLineEditor) {
+                true // show it
+            } else {
+                missing_editors.push(component);
+                false // don't shopw
+            }
+        });
+
+        if component_to_vis.is_empty() && disabled_reason.is_none() {
+            disabled_reason = Some(format!(
+                "Rerun lacks {} for: {}",
+                if missing_editors.len() == 1 {
+                    "an editor"
+                } else {
+                    "editors"
+                },
+                missing_editors.iter().map(|c| c.short_name()).join(" ")
```

Comment:
status quo is that this might be a lot actually 🤔. Ah well, probably fine, there's an obvious solution to that 😄 (add more editors!)

Reply:
```suggestion
                missing_editors.iter().map(|c| c.short_name()).join(", ")
```
Comma reads nicer imho